### ISSUE TITLE
Flipflop dashboard table could use some accessibility improvements

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3562]
### Issues

- The first column (the "status" column) does not have any text in the header.  While navigating with a screen reader, it would be helpful if it referred to this column as "status", rather than simply "column 1 of 6".  When I first started using flipflop, understanding this column tripped me up even without using a screen reader.
- The "Feature" column essentially acts as a row header.  It would be helpful if it were marked up as a row header, so that the screen reader could announce, say, "New action note display", rather than simply "row 3 of 4".

### Steps to recreate

1. Log in
3. Navigate to /features
1. Start voice over
1. Press ctrl + option + U to open the rotor
2. Use the arrow keys to find the table list
3. Press enter to select the table
4. Press ctrl + option + shift + down arrow to enter the table
5. Use ctrl + option + arrow keys to navigate to various table cells and hear how voiceover announces them.  Expected behavior is that when you navigate to a different column, voiceover will announce the new column's header.  When you navigate to a different row, voiceover will announce the new row's header.

### URL

https://catalog.princeton.edu/features

### Severity

Low severity (despite its conformance level) -- it's possible to figure this out from context, just a little annoying.

### WCAG Conformance level (A, AA, AAA)

Level A

### User impact

Low.  Very few users access the flipflop dashboard.

### Recommended fix

Open a PR [upstream](https://github.com/voormedia/flipflop) (or multiple PRs) that does the following:

- [ ] adds text to the first column ("Status" column)
- [ ] marks up the "Feature" column as a row header

### Acceptance criteria
- [ ] When announcing a table cell, Voice Over in Safari offers helpful information about which column and row it is in, rather than just "row 1 of 3, column 3 of 6"
- [ ] WAVE no longer lists an "Empty table header" error
- [ ] aXe devtools no longer lists a "Table header text should not be empty" error when best practices are turned on

### Notes

- Also affects lockers and soon, lib-jobs.
- WebAIM has [good documentation about accessible tables](https://webaim.org/techniques/tables/data).
- This would be a good ticket for anybody who wants to learn more about accessible tables.  I didn't add the good first issue label, though, since it involves a non-PUL project.
